### PR TITLE
Renaming and rearranging transaction-related items

### PR DESCRIPTION
**Pull request by viktorprogger**

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: src/Domain/Operation/TransactionCreationData.php [hunk 1]
```diff
@@ -7,22 +7,22 @@
...
-    public AccountIdInterface $debitAccountId;
     public AccountIdInterface $creditAccountId;
+    public AccountIdInterface $debitAccountId;
     public Money $amount;
 
     public function __construct(
         TransactionIdInterface $id,
-        AccountIdInterface $debitAccountId,
         AccountIdInterface $creditAccountId,
+        AccountIdInterface $debitAccountId,
```
Comment (vjik) [inline src/Domain/Operation/TransactionCreationData.php:20]: Does order (debit/credit or credit/debit) it matted?

Comment (viktorprogger) [reply src/Domain/Operation/TransactionCreationData.php:20]: In natural language we usually say "From point A to point B". This means, resource point (credit) goes first. In my opinion, the same order in function arguments is more intuitive.

**Comment by vjik** [comment]
Ok, for me equal :)

**Comment by vjik** [comment]
Ok, for me equal :)

File: src/Domain/Operation/TransactionCreationData.php (hunk 1 above)
Comment (vjik) [reply src/Domain/Operation/TransactionCreationData.php:20]: Ok, for me equal :)

**Review by vjik — APPROVED** [review]

**Pull request merged**